packages/router-bridge/src/sagaRouter/module.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { Router } from 'react-router-dom';
+
+import { connectRouter, routerMiddleware, ConnectedRouter } from 'connected-react-router';
+import { fork, takeLatest } from 'redux-saga/effects';
+
+import cmf from '@talend/react-cmf';
+import { sagaRouter, SAGA_ROUTER_HISTORY_CHANGE } from './sagaRouter';
+
+const { history }: { history: any } = require('../router');
+
+const mergeConfig = {
+	history: cmf.module.merge.getUnique,
```

Comment:
This isn't used here, the history that is used is the one created by router bridge, which has to be unique